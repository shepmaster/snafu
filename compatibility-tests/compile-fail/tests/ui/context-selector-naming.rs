use snafu::prelude::*;

// FIXME: https://github.com/shepmaster/snafu/issues/486 -- needs semver bump
// #[derive(Debug, Snafu)]
// #[snafu(context)]
// struct CannotUseUnqualifiedContext;

// FIXME: https://github.com/shepmaster/snafu/issues/486 -- needs semver bump
// #[derive(Debug, Snafu)]
// #[snafu(context(true))]
// struct CannotUseContextTrue;

#[derive(Debug, Snafu)]
#[snafu(context(name(Bob), suffix(X)))]
struct CannotUseBothNameAndSuffix;

fn main() {}
