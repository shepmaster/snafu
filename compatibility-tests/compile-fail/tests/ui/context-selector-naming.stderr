error: unexpected token, expected `)`
  --> tests/ui/context-selector-naming.rs:14:26
   |
14 | #[snafu(context(name(Bob), suffix(X)))]
   |                          ^
